modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/api/URLHelper.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.apidoc.api;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * Helper for URL validity checks.
+ *
+ * @since 20.2.0
+ */
+public class URLHelper {
+
+    public static boolean isValid(String url) {
+        return isValid(url, true);
+    }
+
+    protected static boolean isValid(String url, boolean redirect) {
+        try {
+            HttpURLConnection huc = (HttpURLConnection) new URL(url).openConnection();
+            if (!redirect) {
+                huc.setInstanceFollowRedirects(false);
+            }
```

Comment:
Why not the below?
```suggestion
            huc.setInstanceFollowRedirects(redirect);
```
Furthermore, why do we need the `redirect` parameter? I don't see any `false` usage.

Reply:
There isn't any right now, but since the marketplace link check does not require it, i was thinking to have a `false` usage there... does not seem useful after all indeed